runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -569,6 +576,10 @@ else if (config.swapTeleportItem() && option.equals("wear"))
 			swap("rub", option, target, true);
 			swap("teleport", option, target, true);
 		}
+		else if (config.swapMtaItems() && MTA_ITEMS.contains(eventId) && ("wield".equals(option) || "wear".equals(option)))
```

Comment:
I guess it should be `(option.equals("wield") || option.equals("wear"))` for consistency with the other conditional checks in this chain. My bad for recommending the other way around first.